cmd/crates/stellar-ledger/src/emulator_tests.rs

```diff
@@ -23,13 +21,35 @@ fn ledger(host_port: u16) -> LedgerSigner<impl Exchange> {
     LedgerSigner::new(get_http_transport("127.0.0.1", host_port).unwrap())
 }
 
+mod test_helpers {
+    pub mod test {
+        include!("../tests/test/mod.rs");
+    }
+}
+
+use test_helpers::test::{
+    emulator_http_transport::EmulatorHttpTransport,
+    speculos::{Args, Speculos},
+};
+
 #[tokio::test]
-async fn test_get_public_key() {
+async fn test_get_public_key_for_different_ledger_devices() {
```

Comment:
instead of having these in a loop in one test, i am thinking that i may make sense to break each device out into its own test so that they will be run in paralell